sympy/functions/elementary/exponential.py

```diff
@@ -1011,19 +1012,18 @@ def coeff_exp(term, x):
 
         # TODO new and probably slow
         try:
-            _, b = z.leadterm(t, logx=logx, cdir=1)
-            s = z._eval_nseries(t, n=n+b, logx=logx, cdir=1)
-        except (ValueError, NotImplementedError, PoleError):
-            s = z._eval_nseries(t, n=n, logx=logx, cdir=1)
+            a, b = z.leadterm(t, logx=logx if _logx else _logx, cdir=1)
+        except:
```

Comment:
Not all errors should be caught.

Reply:
Yeah I had changed it to (`ValueError, NotImplementedError, PoleError`) in the latest commit ! Not sure if you missed that , maybe you could refresh ?